packages/common/src/discussions/api/types.ts

```diff
@@ -510,7 +510,16 @@ export interface ICreatePost extends IPostOptions {
  * @extends {IPostOptions}
  * @extends {ICreateChannel}
  */
-export interface ICreateChannelPost extends IPostOptions, ICreateChannel {}
+export interface ICreateChannelPost
```

Comment:
The interface is used in V1 create post (with channel create on the fly)

`name` and `channelAclDefinition` required in `ICreateChannel` for V2, not required for V1
`channelAclDefinition` added to the body by the API during V1 -> V2 conversion, so needs to be on the dto